python/paddle/v2/framework/layers.py

```diff
@@ -22,12 +22,36 @@ def fc(input,
        num_flatten_dims=1,
        main_program=None,
        startup_program=None):
-    # create helper
+    """
+    Fully Connected Layer.
+
+    Args:
+       input: The input tensor to the function
+       size: The size of the layer
+       param_attr: The parameters/weights to the FC Layer
```

Comment:
Need to document what are the parameters for param_attr and bias_attr somewhere.

Reply:
Could you elaborate more on what parameters for param_attr would mean ?